hoomd/dem/DEM3DForceCompute.cc

```diff
@@ -60,6 +60,66 @@ DEM3DForceCompute<Real, Real4, Potential>::DEM3DForceCompute(
         }
     }
 
+template<typename Real, typename Real4, typename Potential>
+void DEM3DForceCompute<Real, Real4, Potential>::connectDEMGSDShapeSpec(std::shared_ptr<GSDDumpWriter> writer)
+    {
+    typedef hoomd::detail::SharedSignalSlot<int(gsd_handle&)> SlotType;
+    auto func = std::bind(&DEM3DForceCompute<Real, Real4, Potential>::slotWriteDEMGSDShapeSpec, this, std::placeholders::_1);
+    std::shared_ptr<hoomd::detail::SignalSlot> pslot(new SlotType(writer->getWriteSignal(), func));
+    addSlot(pslot);
+    }
+
+template<typename Real, typename Real4, typename Potential>
+int DEM3DForceCompute<Real, Real4, Potential>::slotWriteDEMGSDShapeSpec(gsd_handle& handle) const
+    {
+    GSDShapeSpecWriter shapespec(m_exec_conf);
+    m_exec_conf->msg->notice(10) << "DEM3DForceCompute writing to GSD File to name: " << shapespec.getName() << std::endl;
```

Comment:
This log message doesn't make sense to me. Perhaps `DEM3DForceCompute writing particle shape information to GSD file in chunk ...`